contracts/upgradeable_contracts/erc20_to_native/HomeBridgeErcToNative.sol

```diff
@@ -16,12 +16,17 @@ contract HomeBridgeErcToNative is EternalStorage, BasicBridge, BasicHomeBridge,
     event AmountLimitExceeded(address recipient, uint256 value, bytes32 transactionHash);
 
     function () public payable {
+        nativeTransfer();
+    }
+
+    function nativeTransfer() public payable {
```

Comment:
Should it be `public` indeed? Consider to make it internal.

Reply:
Agree, this should be internal https://github.com/poanetwork/poa-bridge-contracts/pull/203/commits/2ab1e7dedcdb3c1e9c3981c13ffb902b8c3ce4cc